File: usermanagement/src/main/java/md/utm/fcim/usermanagement/config/jersey/JerseyConfig.java
package md.utm.fcim.usermanagement.config.jersey;

import md.utm.fcim.usermanagement.webservice.impl.UserControllerImpl;
import org.glassfish.jersey.server.ResourceConfig;
import org.springframework.stereotype.Component;

@Component
public class JerseyConfig extends ResourceConfig {

    public JerseyConfig() {
        registerEndpoints();
    }

    private void registerEndpoints() {
        register(UserControllerImpl.class);
    }

}

File: usermanagement/src/main/kotlin/md/utm/fcim/usermanagement/config/jersey/JerseyConfig.kt
package md.utm.fcim.usermanagement.config.jersey

import md.utm.fcim.usermanagement.webservice.impl.UserControllerImpl
import org.glassfish.jersey.server.ResourceConfig
import org.springframework.stereotype.Component

@Component
class JerseyConfig : ResourceConfig() {
    init {
        registerEndpoints()
    }

    private final fun registerEndpoints() {
        register(UserControllerImpl::class.java)
    }

}
